plutus-pab/src/Plutus/PAB/Run/PSGenerator.hs

```diff
@@ -150,8 +151,8 @@ generateAPIModule _ outputDir = do
         mySettings
         outputDir
         pabBridgeProxy
-        (    Proxy @(API.API (Builtin a)
-        :<|> API.NewAPI (Builtin a) Text.Text
+        (    Proxy @(API.API (Contract.ContractDef (Builtin a))
+        :<|> API.NewAPI (Contract.ContractDef (Builtin a)) Text.Text
```

Comment:
`ContractDef (Builtin a)` is basically just `a`. I _think_ you can just change it to:

```suggestion
        ( Proxy @(API.API a :<|> API.NewAPI a Text.Text
```

Reply:
The saving of lines is tempting :), but in case the definition changes hopefully this helps catch it sooner